### PR TITLE
doc: block-relay-only vs blocksonly

### PR DESCRIPTION
**Pull request by maflcko**
Those are different concepts, see https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.19.0.1.md#p2p-changes for the block-relay-only nodes.

### CODE REVIEW

File: doc/reduce-traffic.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Some node operators need to deal with bandwidth caps imposed by their ISPs.
 By default, Bitcoin Core allows up to 125 connections to different peers, 10 of
 which are outbound. You can therefore, have at most 115 inbound connections.
 Of the 10 outbound peers, there can be 8 full outgoing connections and 2 with
```
Comment (jonatack) [inline doc/reduce-traffic.md]: suggest s/full outgoing/full-relay/ as the sentence already stated that these are outbound and the codebase (net.h/net.cpp/net_processing.cpp) and debug log call them full-relay

File: doc/reduce-memory.md [hunk 2]
```diff
@@ -24,7 +24,9 @@ The size of some in-memory caches can be reduced. As caches trade off memory usa
 
 ## Number of peers
 
-- `-maxconnections=<n>` - the maximum number of connections, this defaults to `125`. Each active connection takes up some memory. Only significant if incoming connections are enabled, otherwise the number of connections will never be more than `10`. Of the 10 outbound peers, there can be 8 full outgoing connections and 2 -blocksonly peers, in which case they are block/addr peers, but not tx peers.
+- `-maxconnections=<n>` - the maximum number of connections, this defaults to `125`. Each active connection takes up
+  some memory. Only significant if incoming connections are enabled, otherwise the number of connections will never be
```
Comment (jonatack) [inline doc/reduce-memory.md]: suggest clarifying with s/Only significant/This option applies only/ or something similar

(as is, it reads almost like the memory use is only significant if...)

File: doc/reduce-memory.md [hunk 3]
```diff
@@ -24,7 +24,9 @@ The size of some in-memory caches can be reduced. As caches trade off memory usa
...
+  more than `10`. Of the 10 outbound peers, there can be 8 full outgoing connections and 2 block-relay-only peers.
```
Comment (jonatack) [inline doc/reduce-memory.md]: not sure why 125 and 10 are enclosed with `code` markup

**Review by jonatack — COMMENTED** [review]
ACK with a few additional suggestions

File: doc/reduce-traffic.md (hunk 1 above)
Comment (MarcoFalke) [reply doc/reduce-traffic.md]: Fixed both occurrences

**Comment by MarcoFalke** [comment]
Addressed feedback by @jonatack

**Comment by jonatack** [comment]
ACK fae5d88

File: doc/reduce-memory.md [hunk 4]
```diff
@@ -24,7 +24,9 @@ The size of some in-memory caches can be reduced. As caches trade off memory usa
 
 ## Number of peers
 
-- `-maxconnections=<n>` - the maximum number of connections, this defaults to `125`. Each active connection takes up some memory. Only significant if incoming connections are enabled, otherwise the number of connections will never be more than `10`. Of the 10 outbound peers, there can be 8 full outgoing connections and 2 -blocksonly peers, in which case they are block/addr peers, but not tx peers.
+- `-maxconnections=<n>` - the maximum number of connections, this defaults to 125. Each active connection takes up some
+  memory. This option applies only if incoming connections are enabled, otherwise the number of connections will never
+  be more than 10. Of the 10 outbound peers, there can be 8 full-relay connections and 2 block-relay-only peers.
```
Comment (hebasto) [inline doc/reduce-memory.md]:
```suggestion
  be more than 10. Of the 10 outbound peers, there can be 8 full-relay connections and 2 block-relay-only ones.
```

File: doc/reduce-traffic.md [hunk 5]
```diff
@@ -5,8 +5,8 @@ Some node operators need to deal with bandwidth caps imposed by their ISPs.
 
 By default, Bitcoin Core allows up to 125 connections to different peers, 10 of
 which are outbound. You can therefore, have at most 115 inbound connections.
-Of the 10 outbound peers, there can be 8 full outgoing connections and 2 with
-the -blocksonly mode turned on. You can therefore, have at most 115 inbound connections.
+Of the 10 outbound peers, there can be 8 full-relay connections and 2 with
+block-relay only.
```
Comment (hebasto) [inline doc/reduce-traffic.md]:
```suggestion
Of the 10 outbound peers, there can be 8 full-relay connections and 2
block-relay-only ones.
```

**Review by hebasto — COMMENTED** [review]
I assume that "block-relay-only" should be applied to the "connection", but not to the "peer".

**Review by hebasto — APPROVED** [review]
ACK fa6e01f2a163511a735088895ab02232b150801b

**Comment by jonatack** [comment]
ACK fa6e01f

pico-nit: as the outbound peers are called "full-relay" and "block-relay" in the debug log, perhaps use "block-relay" rather than "block-relay-only". Though the code comments use block-relay-only, the -only seems redundant.

File: doc/reduce-memory.md (hunk 4 above)
Comment (jonatack) [reply doc/reduce-memory.md]: Agree, this is better.

**Pull request merged**